src/main/scala/com/github/tototoshi/csv/CSVReader.scala

```diff
@@ -132,10 +132,21 @@ object CSVReader {
     }
   }
 
+  def open(file: File, encoding: String)(implicit format: CSVFormat): CSVReader = {
```

Comment:
About implementation, I think it's better to reuse `open(file: File, encoding: Charset)`

```scala
open(file, Charset.forName(encoding))
```